controller/setting_controller.go

```diff
@@ -186,6 +188,11 @@ func (sc *SettingController) processNextWorkItem() bool {
 	defer sc.queue.Done(key)
 
 	err := sc.syncSetting(key.(string))
+	if err != nil && strings.Contains(err.Error(), lhWorkloadAttachedVolumeMsg) {
+		sc.logger.WithError(err).Debugf("Retrying to update setting %v", key)
+		sc.queue.AddRateLimited(key)
```

Comment:
What happens if the setting has been changed several times without ever detaching all volumes?

Reply:
Without ever detaching all volumes, it should always retry to update with the latest changed value.

Removed this part. Thanks.